commcare_connect/templates/opportunity/user_visit_details.html

```diff
@@ -175,32 +175,43 @@
   {% endif %}
 
   <!-- Verification Flags -->
-  <div class="grid grid-cols-3 gap-4">
-    <p class="text-sm font-medium text-brand-deep-purple col-span-3">Verification flags</p>
-    {% if user_visit.location %}
-    <div>
-      <div class="text-sm text-brand-blue-light">Location</div>
-      <div class="text-sm font-normal text-brand-deep-purple">Less than {{ closest_distance }} metres</div>
+  <div class="flex flex-col gap-4">
+    <p class="text-sm font-medium text-brand-deep-purple">Verification Parameters</p>
+    <div class="grid grid-cols-3 gap-4">
+      {% if user_visit.location %}
+      <div>
+        <div class="text-sm text-brand-blue-light">Location</div>
+        <div class="text-sm font-normal text-brand-deep-purple">Less than {{ closest_distance }} metres</div>
```

Comment:
nit, this should be tagged for translation, as well as a few other places in this page